tools/python/translate.py

```diff
@@ -82,7 +82,7 @@ def google_translate(text):
   fromTo = fromTo[:-1]
   res = subprocess.run([TRANS_CMD, '-b', '-no-bidi', fromTo, text], text=True, capture_output=True)
   if res.returncode != 0:
-    print('Error running trans program:')
+    print(f'Error running {TRANS_CMD} program:')
```

Comment:
Is it worth it?

Reply:
Or what do you mean by "is it worth it"? :)
Its not related to python >= 3.7 requirement, which comes from https://github.com/organicmaps/organicmaps/pull/5228

Actually, I'm not sure its a good idea to raise the minimum version requirement because an older version had a security bug... I think a min version requirement is more about API compatibility. A user is supposed to fetch the latest version anyway and its supposed to have the latest fixes. CC @biodranik